data/tests/test_arango_db.py

```diff
@@ -170,7 +170,7 @@ def test_arangodb_generates_import_statements_for_nodes(mock_client):
         db = ArangoDB()
 
         cmds = db.generate_import_statement(
-            'header-test.csv', ['data1.csv', 'data2.csv'], 'test_collection', 'node')
+            'header-test.csv', ['data1.csv', 'data2.csv'], 'test_collection', 'node', True)
 
```

Comment:
we need an extra test with False just to guarantee the `--translate` parameter won't be passed

Reply:
thank you 🙏